src/screens/Login/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { isEmpty } from 'lodash';
+import validator from 'validator';
+
+import Alert from 'components/Alert';
+import Button from 'components/Button';
+import WarningIcon from 'components/Icon/WarningIcon';
+import Input from 'components/Input';
+import AuthLayout from 'components/Layout/Auth';
+
+const LoginScreen = (): JSX.Element => {
+  const { t } = useTranslation();
+  const [formInput, setformInput] = useState({ email: '', password: '' });
+  const [errorMessage, setErrorMessage] = useState<string[]>([]);
+  const [formLoading, setFormLoading] = useState(false);
+  const validateInput = (input: { email: string; password: string }) => {
+    const errors = [];
+    if (!input.email) {
+      errors.push(t('error.email_blank'));
+    }
+    if (!input.password) {
+      errors.push(t('error.password_blank'));
+    }
+    if (!validator.isEmail(input.email)) {
+      errors.push(t('error.email_invalid'));
+    }
+    setErrorMessage(errors);
+  };
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setFormLoading(true);
+    try {
+      validateInput(formInput);
+    } catch (error) {
+      console.log('error: ', error);
+    }
+    setFormLoading(false);
+  };
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setformInput({ ...formInput, [event.target.name]: event.target.value });
+  };
+
+  return (
+    <AuthLayout headerMessage={t('auth.heading')}>
+      <div className="form-group">
+        {errorMessage && !isEmpty(errorMessage) && <Alert Icon={WarningIcon} errorMessage={errorMessage}></Alert>}
+        <form onSubmit={handleSubmit}>
+          <Input label={t('auth.sign_in')} onChange={handleChange} type="text" name="email"></Input>
```

Comment:
Should the type be `email`?
```suggestion
          <Input label={t('auth.sign_in')} onChange={handleChange} type="email" name="email"></Input>
```

Reply:
Sure actually I have a library to validate the email but this one is better. Fixed in this [commit](https://github.com/suphanatjarukulgowit/react-nimble-survey/commit/f5c1469821f5cd51510a802528ec903b4dd6df28#diff-8803b1e70e968385dc7b2d28d7db1cb9c2ce3c2b54d0b99cedc5882468cdc4ba) 👍